lib/ADC/ADC.h

```diff
@@ -8,6 +8,8 @@
 #include <ADS1X15.h>
 #include <definitions.h>
 
+#define ADC_MAX 65535
```

Comment:
Maybe move into the class or add to `definitions.h`?

Reply:
moved in definitions.h